### PR TITLE
Integrate ele

### PR DESCRIPTION
**Pull request by daico007**
### PR Summary:

Remove parmed element dependency in the atomtyping step. Relate to #391

### PR Checklist
------------
 - [ ] Includes appropriate unit test(s)
 - [ ] Appropriate docstring(s) are added/updated
 - [ ] Code is (approximately) PEP8 compliant
 - [ ] Issue(s) raised/addressed?

### CODE REVIEW

File: foyer/atomtyper.py [hunk 1]
```diff
@@ -48,8 +48,8 @@ def find_atomtypes(structure, forcefield, max_iter=10):
                 system_elements.add(name)
         else:
             atomic_number = atom_data.atomic_number
-            if 0 < atomic_number <= pt.KNOWN_ELEMENTS:
-                element = pt.Element[atomic_number]
+            if 0 < atomic_number <= len(ele.Elements[0]):
```
Comment (rsdefever) [inline foyer/atomtyper.py:51]: Why not something like:

```python
try:
    element = ele.element_from_atomic_number(atomic_number).symbol
except ElementError:
    raise FoyerError(...)
```
```

**Comment by umesh-timalsina** [comment]
While we are at it can we also take care of https://github.com/mosdef-hub/foyer/blob/a3ec0177507d847ec53136d8c49f424ce188f8a1/foyer/forcefield.py#L16-L17 imports here and remove `foyer/element.py`

**Comment by daico007** [comment]
> While we are at it can we also take care of
> 
> https://github.com/mosdef-hub/foyer/blob/a3ec0177507d847ec53136d8c49f424ce188f8a1/foyer/forcefield.py#L16-L17
> 
> imports here and remove `foyer/element.py`

I think the we still need the OpenMM `elem` here, they are used to infer OpenMM Element objects. I don't think the OpenMM forcefield will accept 3rd party Element object.

File: foyer/atomtyper.py (hunk 1 above)
Comment (rsdefever) [reply foyer/atomtyper.py:51]: @daico007 are you planning on changing this? Your test right now assumes there are no "skipped" elements in the list from `ele`. Ele also provides the `ElementError` for exactly this purpose.

Comment (daico007) [reply foyer/atomtyper.py:51]: Sorry, I missed this comment. I will address this in the next commit.

**Review by justinGilmer — APPROVED** [review]
LGTM!

**Comment by justinGilmer** [comment]
Also, the RTD build has failed. It seems to be missing ele? Although im not entirely sure yet.

**Comment by daico007** [comment]
I added `ele` to the docs build yml, let see if that fixes the issue

**Pull request closed without merging**